src/sagemaker/__init__.py

```diff
@@ -39,4 +41,4 @@
 from sagemaker.session import s3_input  # noqa: F401
 from sagemaker.session import get_execution_role  # noqa: F401
 
-__version__ = '1.18.1'
+__version__ = pkg_resources.require('sagemaker')[0].version
```

Comment:
Not sure if this is the expected behavior. Because the version returned from pkg_resources would be the version of sagemaker installed in the environment instead of the version in setup.py.

Reply:
talked offline - returning the version that's installed should be expected behavior.